front/components/IdeaContent.vue

```diff
@@ -10,6 +11,17 @@ export default {
     content: {
       type: String,
       default: ''
+    },
+
+    preview: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  computed: {
+    strippedContent() {
+      return this.content.replace(/(<([^>]+)>)/gi, '')
```

Comment:
i wonder if it will be heavy for client e.g. mobile to do that. Maybe we should store stripped version on backend

Reply:
That would be way nicer. I'll do that.

Reply:
and since we use graphql we can only request this field on index page and not on other so it wont affect sizes and even make it smaller since normal full html wont be loaded on index